website/static/js/pages/project-base-page.js

```diff
@@ -26,34 +27,34 @@ if (node.isPublic && node.piwikSiteID) {
 $(window).unload(function(){
     return 'Unload';
 });
+
 $(document).ready(function() {
-    $.getJSON(node.urls.api, function(data) {    
+    $.getJSON(node.urls.api, function(data) {
         $('body').trigger('nodeLoad', data);
     });
+});
 
-    var self = this;
-    var THRESHOLD_SCROLL_POSITION  = 50;
-    var SMALL_SCREEN_SIZE = 767;
-    var NON_NAV_TOP_MARGIN = 50;
-    var NAV_MAX_TOP_MARGIN = 95;
-    self.adjustPanelPosition = function() {
-        var bodyWidth = $(document.body).width();
-        var scrollTopPosition = $(window).scrollTop();
-        if (bodyWidth <= SMALL_SCREEN_SIZE) {
-            if (scrollTopPosition >= THRESHOLD_SCROLL_POSITION) {
-                $('.cp-handle').css('margin-top', NON_NAV_TOP_MARGIN);
-            }
-            else {
-                $('.cp-handle').css('margin-top', NAV_MAX_TOP_MARGIN - scrollTopPosition);
-            }
-        } else {
-            $('.cp-handle').css('margin-top', NAV_MAX_TOP_MARGIN);
-        }
-    };
-    var ADJUST_PANEL_DEBOUNCE = 10;
-    var RESIZE_DEBOUNCE  = 50;
-
-    self.adjustPanelPosition(); /* Init when refreshing the page*/
-    $(window).scroll($osf.debounce(self.adjustPanelPosition, ADJUST_PANEL_DEBOUNCE));
-    $(window).resize($osf.debounce(self.adjustPanelPosition, RESIZE_DEBOUNCE));
+$(window).scroll(function() {
```

Comment:
When the comment pane is open on small screen sizes (when the project navigation menu is collapsed) and you scroll down the page there is a gap where the nav bar used to be. This code moves the comment panel to be in line with the nav bar. 

![screen shot 2016-01-04 at 10 57 39 am](https://cloud.githubusercontent.com/assets/6414394/12093630/1251e504-b2d2-11e5-871e-6fa944165551.png)

![screen shot 2016-01-04 at 10 52 24 am](https://cloud.githubusercontent.com/assets/6414394/12093542/8a6d9aa2-b2d1-11e5-9276-0adef2e145fe.png)